### DISCUSSION TITLE
Restic documentation missing for latest version

### DISCUSSION THREAD

**Discussion by VickyWinner** [vmware-tanzu/velero#5639] [Community support Q&A] [answered]
Hi
     When I search for restic documentation, the link takes me to older velero version.  https://velero.io/docs/v1.0.0/restic/ with a message like below.
![image](https://user-images.githubusercontent.com/33285331/204335223-b74047a8-5c0a-4242-a4c9-642900d8bdfe.png)

when I click on the link, I get page not found.

**Comment by sseago** [comment, marked as answer]
What you're looking for is this, I think: https://velero.io/docs/v1.10/file-system-backup/

v1.1.0 is over 3 years old, so quite a bit has been moved around in the documentation since then.
